packages/testsweets/lib/src/models/application_models.dart

```diff
@@ -15,47 +15,37 @@ class WidgetDescription with _$WidgetDescription {
     String? id,
 
     /// The name of the view this widget was captured on
-    required String viewName,
+    String? viewName,
 
     /// The orignal name of the view this widget was captured on before the prettify
-    required String originalViewName,
+    String? originalViewName,
 
     /// The name we want to use when referring to the widget in the scripts
-    required String name,
+    @Default('') String name,
 
     /// The type of the widget that's being added
-    required WidgetType widgetType,
+    WidgetType? widgetType,
 
     /// The position we defined for he widget
-    required WidgetPosition position,
+    WidgetPosition? position,
+
+    /// Whether the key will be visible to the driver or not
+    @Default(true) bool visibility,
   }) = _WidgetDescription;
   factory WidgetDescription.addView(
```

Comment:
This function should be called `view`. This is not adding a view, it's constructing a view. Named constructors should define what you are creating, not what you will do with the result of what you are creating.

Reply:
👍